app/src/main/java/com/celzero/bravedns/ui/DNSConfigureWebViewActivity.kt

```diff
@@ -232,7 +232,7 @@ class DNSConfigureWebViewActivity : AppCompatActivity(R.layout.activity_faq_webv
                 }
```

Comment:
Log the exception to `Log.e(tag, msg, ex)` do not eat it up.

Reply:
Added the exception log.